alacritty_terminal/src/grid/storage.rs

```diff
@@ -813,7 +832,7 @@ mod test {
         };
 
         // Initialize additional lines
-        storage.initialize(3, Row::new(Column(1), &'-'));
+        storage.initialize_exact(3, Row::new(Column(1), &'-'));
```

Comment:
This is the `initialize` test. Please rename this test, and add a new test for the behavior of the new `initialize` function.

Reply:
Something like:

```rust
    #[test]
    fn initialize() {
        // Setup storage area
        let mut storage = Storage {
            inner: vec![],
            zero: 0,
            visible_lines: Line(0),
            len: 0,
        };

        storage.initialize(3, Row::new(Column(0), &' '));

        assert_eq!(3, storage.len);
        assert_eq!(1_000, storage.inner.len());

        storage.initialize(3, Row::new(Column(0), &' '));

        assert_eq!(6, storage.len);
        assert_eq!(1_000, storage.inner.len());
    }
```

Reply:
Well, this test make sense.